jina/helper.py

```diff
@@ -282,12 +282,25 @@ def random_name() -> str:
     return '_'.join(random.choice(_random_names[j]) for j in range(2))
 
 
+_used_ports = set()
```

Comment:
this should not remain here, when is this closed?

Reply:
Hmm why not? if we used up all 10000 possible ports, that's bad, but until then it should be fine, no?

Reply:
because this is global. If a daemon is running forever, it will eventually consume all of these?